src/manager.rs

```diff
@@ -140,9 +140,7 @@ impl AuthenticatorTransport for U2FManager {
             status,
             callback,
         };
-        self.tx
-            .send(action)
-            .map_err(|e| AuthenticatorError::from(e))
+        self.tx.send(action).map_err(AuthenticatorError::from)
```

Comment:
`Ok(self.tx.send(action)?)`?

Reply:
uh, yeah, I don't know why my brain doesn't want to just recognize that!